src/networking.c

```diff
@@ -3997,10 +3997,15 @@ void processEventsWhileBlocked(void) {
  * ========================================================================== */
 
 #define IO_THREADS_MAX_NUM 128
+#define CACHE_LINE_SIZE 64
```

Comment:
just for the record, i happened to see a warning like this in test-freebsd, line 2476: [ci-link](https://github.com/enjoy-binbin/redis/runs/8127173935?check_suite_focus=true#step:4:2484)

```
  networking.c:3963:9: warning: 'CACHE_LINE_SIZE' macro redefined [-Wmacro-redefined]
  #define CACHE_LINE_SIZE 64
          ^
  /usr/include/machine/param.h:93:9: note: previous definition is here
  #define CACHE_LINE_SIZE         (1 << CACHE_LINE_SHIFT)
```

Reply:
I suppose we can't rely on that other define.
and i suppose we can't even really assume that the cache line size on the build time machine will be the same one on the runtime machine.

the two alternatives are:
1. keep using our own define, in which case maybe we should prefix it to avoid collision
2. do a runtime detection and add a pile of code to index the array correctly, instead of relying on the compiler.

i suppose option 1 is sufficient

Reply:
what prefix do you like to use, change it to `IO_THREADS_CACHE_LINE_SIZE`?

Reply:
i think REDIS_CACHE_LINE_SIZE, and maybe move it to config.h or server.h (next to NET_IP_STR_LEN).

but let's wait to see if others happen to be aware of something i'm missing.